packages/redux/src/store.ts

```diff
@@ -14,6 +14,7 @@ export interface DeephavenPluginModule {}
 export type DeephavenPluginModuleMap = Map<string, DeephavenPluginModule>;
 
 export interface UserPermissions {
+  isACLEditor: boolean;
   isSuperUser: boolean;
   isQueryViewOnly: boolean;
```

Comment:
Side note - these permissions really only exist/matter for Enterprise users, so we should be extending the interface in Enterprise redux to add these permissions. That hasn't been converted to TS yet though, and it's not doing much harm being added here.